packages/components/MenuButton/src/MenuButton.types.ts

```diff
@@ -7,6 +7,7 @@ export const MenuButtonName = 'MenuButton';
 
 export interface MenuButtonContext {
   showContextualMenu?: boolean;
+  primary?: boolean;
```

Comment:
What's the benefit of putting primary in the MenuButtonContext?

Reply:
I can use it in Render method to choose which Button to render

Reply:
You can access the state in the render method though, which also has the primary prop. Context is good if children need the information to render themselves, but I don't think it's needed in this case. Let me know if there's something I'm missing.

Reply:
I checked, state doesn't have primary prop
![image](https://user-images.githubusercontent.com/11574680/129349503-8aaec181-d03c-415f-b603-d4b0fc09240b.png)
Maybe I did smth wrong, but I can't use it without context. I was thinking to pass it to the root slot props, but as it's React.Fragment I can't pass parameters there. I'll try to find another way

Reply:
Maybe it's because inside line 63 of MenuButton.ts, we are putting `primary` only inside state.context, instead of just as a key in state?

Reply:
Yeah that's what I would think too Just changing to access the primary prop off the state wouldn't be enough, you'll need to fill it out somehow, too, maybe by using a useState hook. 

Actually staring at it some more we probably don't even need it in the state,I think  we could just use userProps?

Reply:
I moved primary prop to root slot.